pleo-oss/file-distributor
# Restructure validation module to follow design of other modules

- Rename, simplify validation functions, module
- Remove invalid types in check module
- Move functions from app module to individual modules
- Rename coreValidation to validation
- Ensure environment importing
- Add missing linebreak
- Simplify module, narrow scopes
- Use simplified types for testing
- Rename functions, types
- Move GitHub calls to git module
- Prefer Either to throwing custom errors for passing data
- Use modified return types in tests
- Format Prettier configuration
- Restructure validation to follow other modules
